src/Sylius/Behat/Context/Api/Admin/ManagingCountriesContext.php

```diff
@@ -0,0 +1,379 @@
+            'code',
+            $province->getCode()
+        ));
+    }
+
+    /**
+     * @Then the province should still be named :province in this country
+     */
+    public function theProvinceShouldStillBeNamedInThisCountry(ProvinceInterface $province): void
+    {
+        /** @var CountryInterface $country */
+        $country = $this->sharedStorage->get('country');
+        Assert::true($this->responseChecker->hasItemWithValue(
+            $this->client->subResourceIndex('provinces', $country->getCode()),
+            'code',
+            $province->getCode()
+        ));
+    }
+
+    /**
+     * @Then I should not be able to choose :countryName
+     */
+    public function iShouldNotBeAbleToChoose(string $countryName): void
+    {
+        $this->client->addRequestData('code', $this->getCountryCodeByName($countryName));
+        $response = $this->client->create();
+        Assert::false(
+            $this->responseChecker->isCreationSuccessful($response),
+            'Country has been created successfully, but it should not'
+        );
+        Assert::same($this->responseChecker->getError($response), 'code: Country ISO code must be unique.');
+    }
+
+    /**
+     * @When /^I want to edit (this country)$/
+     * @When I want to create a new province in country :country
+     */
+    public function iWantToEditThisCountry(CountryInterface $country): void
+    {
+        $this->sharedStorage->set('country', $country);
+        $this->client->buildUpdateRequest($country->getCode());
+    }
+
+    /**
+     * @When I enable it
+     */
+    public function iEnableIt(): void
+    {
+        $this->client->addRequestData('enabled', true);
+    }
+
+    /**
+     * @When I disable it
+     */
+    public function iDisableIt(): void
+    {
+        $this->client->addRequestData('enabled', false);
+    }
+
+    /**
+     * @When I save my changes
+     * @When I try to save changes
+     */
+    public function iSaveMyChanges(): void
+    {
+        $this->client->update();
+    }
+
+    /**
+     * @Then I should be notified that it has been successfully edited
+     */
+    public function iShouldBeNotifiedThatItHasBeenSuccessfullyEdited(): void
+    {
+        Assert::true(
+            $this->responseChecker->isUpdateSuccessful($this->client->getLastResponse()),
+            'Country could not be edited'
+        );
+    }
+
+    /**
+     * @Then /^(this country) should be ([^"]+)$/
+     */
+    public function thisCountryShouldBeDisabled(CountryInterface $country, string $enabled): void
+    {
+        Assert::true(
+            $this->responseChecker->hasValue(
+                $this->client->show($country->getCode()),
+                'enabled',
+                $enabled === 'enabled' ? true : false
+            ),
+            'Country is not disabled'
+        );
+    }
+
+    /**
+     * @Then the code field should be disabled
+     */
+    public function theCodeFieldShouldBeDisabled(): void
```

Comment:
We should go the way as it's done in other contexts (see: https://github.com/Sylius/Sylius/blob/master/src/Sylius/Behat/Context/Api/Admin/ManagingProductOptionsContext.php#L246) - rename this step to `I should not be able to edit its code` to make it less ui-specific and check the error on update request